pom.xml

```diff
@@ -168,8 +168,8 @@ under the License.
 				<groupId>org.apache.maven.plugins</groupId>
```

Comment:
Minor nitpick: the comment should now say <!-- Ensures java 7 compatibility  -->